messageapp/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Dependency directory
+node_modules
+
+# Debug log from npm
+npm-debug.log
+
+# Environment Variables should NEVER be published
+.env
+
+.DS_Store
```

Comment:
Nice, but you have already tracking some .DS_Store files you need to untrack them. Search untrack git file.